test/unit/schedule_array.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
@machisuke what do you think about grouping these YAMLs into a new folder in order to clearly separate test scripts and fixtures? Maybe something like `test/unit/fixtures/schedule_array.yml`

Reply:
@markets 
That's a very good idea!!

Fixed with 91b5fb8